### ISSUE TITLE
UnauthorizedAccess

### ISSUE THREAD

**Issue by ivoloshin** [wk-j/vscode-cake-runner#1]
I get the following error:

```
build.ps1 cannot be loaded because running scripts is disabled on this system. For more     
information, see about_Execution_Policies at http://go.microsoft.com/fwlink/?LinkID=135170.                                           
    + CategoryInfo          : SecurityError: (:) [], ParentContainsErrorRecordException                                               
    + FullyQualifiedErrorId : UnauthorizedAccess
```

I tried setting script policy to RemoteSigned and Unrestricted without luck. Any ideas?

**Comment by wk-j** [comment]
Can you try following command in Cmd

```
powershell -ExecutionPolicy ByPass -File build.ps1 -target <TaskName>
```

If above command working I will add into extension.

**Comment by ivoloshin** [comment]
That worked. Thank you.

**Comment by wk-j** [comment]
@ivoloshin  Fixed, please update to 0.0.5

**Comment by ivoloshin** [comment]
Thanks a bunch!
